_posts/2017-06-08-viewer-authentication-api.html

```diff
@@ -226,150 +126,51 @@ <h6>Response</h6>
-                <th width="100%">Description</th>
-            </tr>
-            <tr>
-                <td><strong>url</strong></td>
-                <td>string</td>
-                <td>REQUIRED</td>
-                <td>URL of the authentication flow entry point</td>
-            </tr>
-            <tr>
-                <td><strong>secret</strong></td>
-                <td>string</td>
-                <td>REQUIRED</td>
-                <td>Secret key</td>
-            </tr>
-            <tr>
-                <td><strong>message</strong></td>
-                <td>string</td>
-                <td>REQUIRED</td>
-                <td>Short text above the button on the screen displayed in the player</td>
-            </tr>
-            <tr>
-                <td><strong>button_caption</strong></td>
-                <td>string</td>
-                <td>REQUIRED</td>
-                <td>Text of the the button on the screen displayed in the player</td>
-            </tr>
-            <tr>
-                <td><strong>popup_width</strong></td>
-                <td>int</td>
-                <td>REQUIRED</td>
-                <td>Width of the popup window with the authenticaion flow</td>
-            </tr>
-            <tr>
-                <td><strong>popup_height</strong></td>
-                <td>int</td>
-                <td>REQUIRED</td>
-                <td>Height of the popup window with the authenticaion flow</td>
-            </tr>
-        </table>
-
-        <h6>Response</h6>
-
-        <p>HTTP response codes returned when the action is successful:</p>
-
-        <table class="table responsive">
-            <tr>
-                <th>HTTP response code</th>
-                <th width="100%">Error conditions</th>
-            </tr>
-            <tr>
-                <td>201 Created</td>
-                <td>Advanced viewer authentication has been set up on the channel with the new hash.</td>
-            </tr>
-            <tr>
-                <td>204 No Content</td>
-                <td>Advanced viewer authentication has been set up on the channel by updating the existing hash with the new one.</td>
-            </tr>
-        </table>
-
-        <p>Specific error codes returned when the action failed:</p>
-
-        <table class="table responsive">
-            <tr>
-                <th>error value</th>
-                <th>HTTP response code</th>
-                <th width="100%">Error conditions</th>
-            </tr>
-            <tr>
-                <td>invalid_request</td>
-                <td>400 Bad Request</td>
-                <td>One or more required parameters are missing or the user has no access to the video.</td>
-            </tr>
-            <tr>
-                <td>invalid_type</td>
-                <td>400 Bad Request</td>
-                <td>The specified type is invalid or unsupported.</td>
-            </tr>
-        </table>
+        <pre><code>http://www.ustream.tv/embed/hashlock/pass?hash="false"</code></pre>
```

Comment:
It should be http://www.ustream.tv/embed/hashlock/pass?hash=false without double quotes
Btw the protocol should match the protocol of the embed's protocol to avoid same-origin policy issues

Reply:
Thanks! 

> Btw the protocol should match the protocol of the embed's protocol to avoid same-origin policy issues

How do you suggest we indicate that? Should I modify it to the protocol-agnostic //www.ustream.tv/... or would that be even more misleading? Should I write http[s]://www.ustream.tv/... maybe?